netmap/netmap_contract.go

```diff
@@ -137,7 +146,7 @@ func Update(script []byte, manifest []byte, data interface{}) {
 //
 // If notary enabled, then look to NeoFSAlphabet role in native RoleManagement
 // contract of the side chain.
-func InnerRingList() []common.IRNode {
+func InnerRingList() []interop.PublicKey {
```

Comment:
While this public method is still a thing (until notary will be enabled in the chains), I propose to keep return value the same.